webui/tests/test-cluster/devcluster.yaml

```diff
@@ -21,9 +30,13 @@ stages:
         root: tools/build
 
   - agent:
-      name: agent1
+      pre:
+        - sh: make -C agent build
       config_file:
         master_host: 127.0.0.1
-        master_port: 8080
-        agent_id: agent1
+        master_port: 8081
         container_master_host: $DOCKER_LOCALHOST
+        agent_id: agent3
+        fluent:
+          port: 24226  # default value is 24224
+          container_name: determined-fluent-3
```

Comment:
suggestion: pin the agent slot type to be CPU to that the tests run on the same* type of accelerator between local and ci  `        slot_type: cpu`